DQM/HcalMonitorTasks/src/HcalDigiMonitor.cc

```diff
@@ -1159,9 +1160,14 @@ int HcalDigiMonitor::process_Digi(DIGI& digi, DigiHists& h, int& firstcap)
   return err;
 } // template <class DIGI> int HcalDigiMonitor::process_Digi
 
+void HcalDigiMonitor::beginRun(const edm::EventSetup& c) {
+  edm::ESHandle<HcalTopology> htopo;
+  c.get<HcalRecNumberingRecord>().get(htopo);
+  topo_  = new HcalTopology(*htopo);
```

Comment:
I forgot to delete this part. This is not required. I shall update the PR by deleting this part and checking if it is OK without this.
________________________________
From: Chris Jones [notifications@github.com]
Sent: 05 October 2015 16:56
To: cms-sw/cmssw
Cc: Sunanda Banerjee
Subject: Re: [cmssw] bsunanda:Run2-hcx34 Avoid the crashes in the DQM code (#11579)


In DQM/HcalMonitorTasks/src/HcalDigiMonitor.cc<https://github.com/cms-sw/cmssw/pull/11579#discussion_r41153225>:

> @@ -1159,9 +1160,14 @@ int HcalDigiMonitor::process_Digi(DIGI& digi, DigiHists& h, int& firstcap)
>    return err;
>  } // template <class DIGI> int HcalDigiMonitor::process_Digi
>
> +void HcalDigiMonitor::beginRun(const edm::EventSetup& c) {
> +  edm::ESHandle<HcalTopology> htopo;
> +  c.get<HcalRecNumberingRecord>().get(htopo);
> +  topo_  = new HcalTopology(*htopo);


Why are you making a copy?

—
Reply to this email directly or view it on GitHub<https://github.com/cms-sw/cmssw/pull/11579/files#r41153225>.